package/oxide/package

```diff
@@ -2,47 +2,50 @@
 # Copyright (c) 2020 The Toltec Contributors
 # SPDX-License-Identifier: MIT
 
-pkgnames=(erode fret oxide rot tarnish)
-pkgver=2.0.3~beta-2
+pkgnames=(erode fret oxide rot tarnish decay)
+pkgver=2.1~beta-1
 timestamp=2021-01-07T03:28Z
 maintainer="Eeems <eeems@eeems.email>"
 license=MIT
 
-source=(https://github.com/Eeems/oxide/releases/download/v2.0.3-beta/oxide.zip)
-sha256sums=(c35f62d33ec954d88fc36fd9853bcb37e54ac45b936364f4014bf6d4b69725e1)
+source=(https://github.com/Eeems/oxide/releases/download/v2.1/oxide.zip)
+sha256sums=(ca5b58a98242cfaf846545c391dca75dc4d13e443764e656cdc621ce07d091f8)
 
 erode() {
     pkgdesc="Task manager"
     url=https://github.com/Eeems/oxide/tree/master/applications/process-manager
     section=utils
-    depends=(tarnish)
+    depends=("tarnish (= $pkgver)")
 
     package() {
         install -D -m 755 -t "$pkgdir"/opt/bin "$srcdir"/opt/bin/erode
+        install -D -m 644 -t "$pkgdir"/opt/usr/share/applications "$srcdir"/opt/usr/share/applications/codes.eeems.erode.oxide
     }
 }
 
 fret() {
     pkgdesc="Take screenshots"
     url=https://github.com/Eeems/oxide/tree/master/applications/screenshot-tool
     section=utils
+    depends=("tarnish (= $pkgver)")
 
     package() {
         install -D -m 755 -t "$pkgdir"/opt/bin "$srcdir"/opt/bin/fret
+        install -D -m 644 -t "$pkgdir"/opt/usr/share/applications "$srcdir"/opt/usr/share/applications/codes.eeems.fret.oxide
     }
 }
 
 oxide() {
     pkgdesc="Launcher application"
     url=https://github.com/Eeems/oxide/tree/master/applications/launcher
     section=launchers
-    depends=(erode fret tarnish xochitl rot)
+    depends=("erode  (= $pkgver)" "fret (= $pkgver)" "tarnish (= $pkgver)" "rot (= $pkgver)" "decay (= $pkgver)")
```

Comment:
What is the reasoning behind moving the `xochitl` dependency to `tarnish`?

Reply:
The oxide package doesn't actually depend on xochitl, tarnish does. Since in theory the oxide package could be provided by other packages in the future (alternative launcher screens) it's better to reflect that. Although perhaps rot should also be moved to tarnish.